portal-frontend/src/app/features/home/inbox/inbox-table/inbox-table.component.ts

```diff
@@ -89,9 +90,9 @@ export class InboxTableComponent implements OnDestroy {
     });
   }
 
-  onRowClick(event: Event) {
-    if (event.currentTarget instanceof HTMLElement) {
-      console.log(event.currentTarget.querySelector<HTMLElement>('td > a')!.click());
```

Comment:
Don't know how I missed this one. I usually do a search for console logs before committing.